R-proj/src/sample_points.cpp

```diff
@@ -236,6 +246,7 @@ Rcpp::NumericMatrix sample_points(Rcpp::Nullable<Rcpp::Reference> P = R_NilValue
                     InnerBall = HP.ComputeInnerBall();
                     if (!set_mean_point) MeanPoint = InnerBall.first;
                 }
+                if (diam < 0.0) diam = 2.0 * std::sqrt(NT(dim)) * InnerBall.second;
```

Comment:
you set this `std::sqrt(NT(dim)) * InnerBall.second` in various places in the code. It is better if you use everywhere `comp_diam()`  So this is set only in one place in the code. That is, if we want to change it in the future we should only change it there. This applies to all types of convex bodies.

Reply:
done